src/js/components/Calendar/Calendar.js

```diff
@@ -132,14 +132,27 @@ const normalizeRange = (value, activeDate) => {
   return range;
 };
 
-const getReference = (reference, value) => {
+const getReference = (reference, value, activeDate) => {
   let nextReference;
   if (value) {
     if (Array.isArray(value)) {
       if (value[0] instanceof Date) {
-        [nextReference] = value;
+        // if we just selected an end date, active date will be 'end'
```

Comment:
Should this read "active date will be 'end'"?

Reply:
fixed